routers/web/explore/repo.go

```diff
@@ -31,7 +31,9 @@ type RepoSearchOptions struct {
 }
 
 // RenderRepoSearch render repositories search page
-func RenderRepoSearch(ctx *context.Context, opts *RepoSearchOptions) {
+// This function is also used to render the Admin Repository Management page.
+// The isAdmin param should be set to true when rendering the Admin page.
+func RenderRepoSearch(ctx *context.Context, opts *RepoSearchOptions, isAdmin bool) {
```

Comment:
Although `isAdmin` looks like an ugly parameter, it can remind the developers that `RenderRepoSearch` is used by the admin page.

Maybe we can avoid mixed use in the future, it's fine for me now.

Reply:
Wouldn't it be better to change the callers of this function to just set the only show relevant option as appropriately? 

Certainly this parameter should really be referencing the fact that it is setting OnlyShowRelevant to false - and thus be named more appropriately.